main.js

```diff
@@ -0,0 +1,2 @@
+let canv = document.querySelector('#Flappy');
+let ctx = cvs.getContext("2d");
```

Comment:
Those lines wont work, you try to access `canvas` with id `Flappy` before it loaded.  Second thing - try using `const` everywhere and change to `let` when necessary.